backend/src/main/java/com/redhat/cloud/notifications/routers/internal/models/AddApplicationRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.redhat.cloud.notifications.routers.internal.models;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.UUID;
+
+public class AddApplicationRequest {
+    @NotNull
+    @Pattern(regexp = "[a-z][a-z_0-9-]*")
+    public String name;
+
+    @NotNull
+    public String displayName;
+
+    @NotNull
+    public UUID bundleId;
+
+    public String ownerRole;
```

Comment:
Is it possible to add a regexp constraint on this?

Reply:
It's not very important but I suppose there are restrictions in RBAC regarding the characters allowed in a role name. I was thinking about doing something similar on our side.

Unrelated: we should put `@Size(max = 200)` on this field since we did it already in `InternalRoleAccess` 😃

Reply:
Unsure about that, but we can check it.
On the plus side, users would only be able to use of their roles. Only us (admins) would be able to type out something here.

Reply:
It's really not important/urgent right now, let's not block this PR because of this 😄